src/vector/backends/numpy.py

```diff
@@ -1424,14 +1424,44 @@ def __repr__(self) -> str:
 
     @property
     def azimuthal(self) -> AzimuthalNumpy:
-        """Returns the azimuthal type class for the given ``VectorNumpy3D`` object."""
-        # TODO: Add an example here - see https://github.com/scikit-hep/vector/issues/194
+        """
+        Returns the azimuthal type class for the given ``VectorNumpy3D`` object.
+
+        Example:
+            >>> import vector
+            >>> vec = vector.array(
+            ... [
+            ...     (1.1, 2.1, 3.1),
+            ...     (1.2, 2.2, 3.2),
+            ...     (1.3, 2.3, 3.3),
+            ...     (1.4, 2.4, 4.4),
+            ...     (1.5, 2.5, 5.5)
+            ...    ], dtype=[("x", float), ("y", float), ("z", float)]
+            ... )
+            >>> print(vec.azimuthal)
```

Comment:
```suggestion
            >>> vec.azimuthal
```
and update the output appropriately to showcase the `repr`!

Reply:
Done!